tests/models/test_dpt.py

```diff
@@ -0,0 +1,276 @@
+        output_tensor_path = hf_hub_download(
+            repo_id=self.hub_checkpoint, filename="output-tensor.pth"
+        )
+
+        input_tensor = torch.load(input_tensor_path, weights_only=True)
+        input_tensor = input_tensor.to(default_device)
+        output_tensor = torch.load(output_tensor_path, weights_only=True)
+        output_tensor = output_tensor.to(default_device)
+
+        with torch.inference_mode():
+            output = model(input_tensor)
+
+        self.assertEqual(output.shape, output_tensor.shape)
+        is_close = torch.allclose(output, output_tensor, atol=5e-2)
+        max_diff = torch.max(torch.abs(output - output_tensor))
+        self.assertTrue(is_close, f"Max diff: {max_diff}")
+
+    @pytest.mark.compile
+    def test_compile(self):
+        if not check_run_test_on_diff_or_main(self.files_for_diff):
+            self.skipTest("No diff and not on `main`.")
+
+        sample = self._get_sample().to(default_device)
+        model = self.get_default_model()
+        model = model.eval().to(default_device)
+
+        if not model._is_torch_compilable:
+            with self.assertRaises(RuntimeError):
+                torch.compiler.reset()
+                compiled_model = torch.compile(
+                    model, fullgraph=True, dynamic=True, backend="eager"
+                )
+            return
+
+        with torch.inference_mode():
+            compiled_model(sample)
+
+    @pytest.mark.torch_export
+    def test_torch_export(self):
+        if not check_run_test_on_diff_or_main(self.files_for_diff):
+            self.skipTest("No diff and not on `main`.")
+
+        sample = self._get_sample().to(default_device)
+        model = self.get_default_model()
+        model.eval()
+
+        exported_model = torch.export.export(
+            model,
+            args=(sample,),
+            strict=True,
+        )
+
+        with torch.inference_mode():
+            eager_output = model(sample)
+            exported_output = exported_model.module().forward(sample)
+
+        self.assertEqual(eager_output.shape, exported_output.shape)
+        torch.testing.assert_close(eager_output, exported_output)
+
+    @pytest.mark.torch_script
+    def test_torch_script(self):
+        if not check_run_test_on_diff_or_main(self.files_for_diff):
+            self.skipTest("No diff and not on `main`.")
+
+        sample = self._get_sample().to(default_device)
+        model = self.get_default_model()
+        model.eval()
+
+        if not model._is_torch_scriptable:
+            with self.assertRaises(RuntimeError):
+                scripted_model = torch.jit.script(model)
+            return
+
+        scripted_model = torch.jit.script(model)
+
+        with torch.inference_mode():
+            scripted_output = scripted_model(sample)
+            eager_output = model(sample)
+
+        self.assertEqual(scripted_output.shape, eager_output.shape)
+        torch.testing.assert_close(scripted_output, eager_output)
```

Comment:
No need to override, please leave only modified tests here, other tests will be fetched from base class